GVFS/GVFS.UnitTests/Platform.Mac/MacGitInstallationTests.cs

```diff
@@ -0,0 +1,18 @@
+﻿using GVFS.Platform.Mac;
+using GVFS.Tests.Should;
+using NUnit.Framework;
+
+namespace GVFS.UnitTests.Platform.Mac
+{
+    [TestFixture]
+    public class MacGitInstallationTests
+    {
+        [TestCase]
+        public void GetInstalledGitBinPathWorks()
+        {
+            string gitBinPath = new MacGitInstallation().GetInstalledGitBinPath();

```

Comment:
> I added a test to exercise this -- it should pass on any VFSForGit running system as git should always be installed, but I have a feeling the unit test will run on Windows too and so will fail. How should I restrict a unit test to Mac(+Linux) only?

VFS4G unit tests should not require that Git be installed and they should be able to run on all platforms (with the exception of the Windows unit tests that require .NET Framework).

If we *really* wanted to unit test the Mac's `GetInstalledGitBinPath` implementation we'd need to mock out the call to `which`.  However, `GetInstalledGitBinPath` is hit quite frequently when cloning/mounting and so I think we're getting sufficient coverage of the method from the functional tests.